lib/modules/datasource/bazel-registry/index.ts

```diff
@@ -0,0 +1,67 @@
+import { ExternalHostError } from '../../../types/errors/external-host-error';
+import { cache } from '../../../util/cache/package/decorator';
+import { HttpError } from '../../../util/http';
+import { BzlmodVersion } from '../../versioning/bazel-module/bzlmod-version';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import type { BazelModuleMetadataResponse } from './types';
+
+export class BazelRegistryDatasource extends Datasource {
+  static readonly id = 'bazel-registry';
```

Comment:
```suggestion
  static readonly id = 'bazel';
```

Is there any problem with making it shorter like this?

Reply:
Not necessarily. The existing `bazel` manager does not use this data source. Would it be confusing to have a datasource with the same name as a manager and not related to each other?

Reply:
In general it's better to have unique names for managers and datasources, but not mandatory. e.g. we have `npm`, `maven` and probably others.

So perhaps it's good to separate them as you originally proposed, the only thing is that it brings some inconsistency as the suffix `-registry` could really apply to most of our datasources, e.g. `orb-registry`, etc.

@JamieMagee @viceice @secustor WDYT?

Reply:
If this will be the standard API used by bazel then I think `bazel` is a better fit.

Reply:
> If this will be the standard API used by bazel then I think `bazel` is a better fit.

agreed

Reply:
Please rename